src/main/kotlin/racingcar/domain/Cars.kt

```diff
@@ -0,0 +1,29 @@
+package racingcar.domain
+
+import java.lang.IllegalArgumentException
+
+class Cars(
+    countOfCars: Int
+) {
+    val values: List<Car>
+    private val gasStation: GasStation = GasStation()
+
+    init {
+        validateMinCount(value = countOfCars)
+        val values = mutableListOf<Car>()
+        for (i in MIN_COUNT..countOfCars) values.add(Car())
+        this.values = values
```

Comment:
kotlin스럽게 아래와 같이 한 줄로 줄여볼 수 있겠네요 :)
```suggestion
        this.values = (MIN_COUNT..countOfCars).map { Car() }
```

Reply:
헉..!! 👍